<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="MacBook-Air-de-Luis.local" failures="1" tests="3" name="pe.edu.galaxy.Invoice" time="15.901" errors="0" timestamp="2020-09-14T21:27:45 PET" skipped="0">
  <testcase classname="pe.edu.galaxy.Invoice" name="goToSingIn" time="7.014"/>
  <system-out/>
  <testcase classname="pe.edu.galaxy.Invoice" name="setCredential" time="3.485"/>
  <system-out/>
  <testcase classname="pe.edu.galaxy.Invoice" name="salePekosita" time="5.402">
    <failure message="expected:&lt;SUNAT: El dato ingresado como unidad de medida no corresponde al valor esperado - Detalle: xxx.xxx.xxx value=&#039;ticket: 1600110682097 error: : 2936: Valor no se encuentra en el catalogo: 03 (nodo: &quot;cbc:InvoicedQuantity/unitCode&quot; valor: &quot;UND&quot;)&#039;&gt; but was:&lt;SUNAT: El dato ingresado como unidad de medida no corresponde al valor esperado - Detalle: xxx.xxx.xxx value=&#039;ticket: 1600136466505 error: : 2936: Valor no se encuentra en el catalogo: 03 (nodo: &quot;cbc:InvoicedQuantity/unitCode&quot; valor: &quot;UND&quot;)&#039;&gt;" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected:<SUNAT: El dato ingresado como unidad de medida no corresponde al valor esperado - Detalle: xxx.xxx.xxx value='ticket: 1600110682097 error: : 2936: Valor no se encuentra en el catalogo: 03 (nodo: "cbc:InvoicedQuantity/unitCode" valor: "UND")'> but was:<SUNAT: El dato ingresado como unidad de medida no corresponde al valor esperado - Detalle: xxx.xxx.xxx value='ticket: 1600136466505 error: : 2936: Valor no se encuentra en el catalogo: 03 (nodo: "cbc:InvoicedQuantity/unitCode" valor: "UND")'>
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:82)
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:87)
at pe.edu.galaxy.Invoice.salePekosita(Invoice.java:86)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- salePekosita -->
  <system-out/>
</testsuite> <!-- pe.edu.galaxy.Invoice -->
